website/src/App.jsx

```diff
@@ -44,6 +44,7 @@ class App extends PureComponent {
             <Route path="/join/class" component={ProtectedJoinClass} />
             <Route path="/dashboard/:optional?" component={ProtectedDashboardRouter} />
             <Route component={NotFound} />
+
```

Comment:
empty line?

Reply:
Forgot about that created a temp view for debugging purposes took it out but forgot to remove the line